cfme/services/myservice/ssui.py

```diff
@@ -261,6 +261,10 @@ class LaunchVMConsole(SSUINavigateStep):
     prerequisite = NavigateToSibling('Details')
```

Comment:
**REQUIRED:**
We should change this instead
```python
def prerequisite(self):
    navigate_to(self, 'Details', wait_for_view=True)
```

Reply:
@mshriver I had to check urban dictionary for TIL 🤣